app/src/Routes/Works/works.json

```diff
@@ -4,16 +4,49 @@
       {
         "title": "Dùn Èideann Blogh",
         "type": "double concerto",
+        "description": "Composed in 2006, written for and dedicated to Robert Walzel, clarinet.  Premiered in the summer of 2007 by the Utah Arts Festival Orchestra, Salt Lake City, with Robert Walzel, clarinet, and Lori Wike, bassoon, Andrew Rindfleisch conductor.",
+        "attribution": "Recorded by The Moravian Philharmonic Orchestra, Robert Walzel, clarinet, Eric Stomberg, bassoon, on <i>Cross Connection</i>; Navona Records, available at <a href=\"https://www.amazon.com/Wolking-Balentine-Connection-Various-artists/dp/B00BLWHTGO\" target=\"_blank\">Amazon</a> and <a href=\"https://itunes.apple.com/us/album/wolking-balentine-cross-connection/id608515694\" target=\"blank\"> iTunes.</a>",
         "instruments": [
-          "clarinet",
-          "bassoon",
-          "orchestra"
+          "Solo Clarinet",
+          "Solo Bassoon",
+          "Single WW: Flute, Oboe, Clarinet, bassoon",
```

Comment:
hopefully stuff like this doesn't look weird because we're already joining things with `, `

Reply:
good point, we may want to change the data to be a bit more consistent